xml/System.Security.Cryptography.X509Certificates/X509VerificationFlags.xml

```diff
@@ -45,10 +45,10 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- These flags indicate the conditions under which chain verification should occur. For example, if an application does not require certificates time values in a chain to be valid, the IgnoreNotTimeValid flag can be used.  
-  
-   
-  
+ These flags indicate the conditions under which chain verification should occur. For example, if an application does not require certificates time values in a chain to be valid, the IgnoreNotTimeValid flag can be used.
+
+ Note that the behavior of certificate validation in .NET Core aims to standardize its behavior according to the underlying win32 implementation APIs (i.e. CertVerifyCertificateChainPolicy in wincrypt.h), even on other platforms. Therefore, AllowUnknownCertificateAuthority not only allows for an untrusted roots, but also permits partial chains on all platforms.
```

Comment:
I don't think this paragraph provides much value, especially the second sentence.

```suggestion
```

Reply:
I think it's important to disclose this piece of information from #49615:

> On Windows, that behavior is controlled by Windows. On non-Windows we maintain parity with what Windows does since that describes the ".NET Framework behavior" for this type, which is the strong starting point for the ".NET behavior" for this type.

This helps provide context for how .NET Core is going to behave on other OSs and how users should expect these flags to behave. AFAIK it's not mentioned anywhere else that the implementation attempts to preserve the behavior of the underlying win32 calls [including quirks like ignoring partial chain, which is what bit me in dotnet/runtime/issues/49615]

Is there a way you'd prefer its phrased?

Reply:
> AFAIK it's not mentioned anywhere else that the implementation attempts to preserve the behavior of the underlying win32 calls

That's generally true of .NET documentation.  We often ended up being a thin wrapper over some Win32 and now that's just "how .NET works".  I don't think it's useful to have in documentation, because it's just a starting point (not all of the OSes we run on can support all of the things in all of the same ways), and it could be interpreted as limiting us from providing additions on top of what Win32 does (e.g. if macOS did something interesting that we wanted to replicate on Windows).

Reply:
OK - Removed from PR